games/devtest/mods/unittests/random.lua

```diff
@@ -0,0 +1,35 @@
+-- test PseudoRandom and PcgRandom state restore api
+
+
+local function test_pseudo_random()
+	local gen1 = PseudoRandom(55)
+	
+	for n=0,128 do
+		gen1:next()
+	end
+	
+	local gen2 = PseudoRandom(gen1:get_state())
+	
+	for n=128,256 do
+		assert(gen1:next()==gen2:next())
+	end
+end
+unittests.register("test_pseudo_random", test_pseudo_random)
```

Comment:
can you move the following code into this file?
<pre>
games/devtest/mods/unittests/misc.lua
1:local function test_random()
2:	-- Try out PseudoRandom
3:	local pseudo = PseudoRandom(13)
4-	assert(pseudo:next() == 22290)
5-	assert(pseudo:next() == 13854)
6-end
7:unittests.register("test_random", test_random)
</pre>

Reply:
Both tests moved to misc. Pseudorandom tests merged.